### PR TITLE
StatusHistory:  Add tooltip multi mode

### PR DESCRIPTION
**Pull request by adela-almasan**
This PR adds tooltip mode "All" to the new Status History tooltip.

Screenshot 2023-11-27 at 11 07 35 AM

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by drew08t** [comment]
Testing it now!

File: public/app/plugins/panel/status-history/StatusHistoryTooltip2.tsx [hunk 1]
```diff
@@ -95,7 +95,6 @@ export const StatusHistoryTooltip2 = ({
         !field ||
         field === xField ||
         field.type === FieldType.time ||
-        field.type !== FieldType.number ||
```
Comment (drew08t) [inline public/app/plugins/panel/status-history/StatusHistoryTooltip2.tsx]: Thanks for fixing the issue I encountered while testing. Are there other types other than time we want to bypass?

Comment (adela-almasan) [reply public/app/plugins/panel/status-history/StatusHistoryTooltip2.tsx]: Looks like not, I was just chatting with @leeoniya about this and it should be ok the way it is.

**Review by drew08t — APPROVED** [review]
LGTM!

**Pull request merged**